src/containers/Authentication/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import Register from "../../components/Register";
+import Login from "../../components/Login";
+import { Form, Row, Button } from "antd";
+import "./index.css";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const Authentication = () => {
+  const [show, setShow] = useState(true);
+  const [click, setClick] = useState(false);
+
+  const handleShow = () => {
+    setShow(true);
+    setClick(false);
+  };
+
+  const handleClick = () => {
+    setClick(true);
+    setShow(false);
+  };
+
+  return (
+    <div>
+      <div className="authButtons">
+        <Form.Item style={{ marginBottom: "5em" }} {...layout} {...tailLayout}>
+          <Row>
+            <Button
+              className="registerButton"
+              type="inner"
+              onClick={handleShow}
+              value="horizontal"
+            >
+              Login
+            </Button>
+            <Button
+              className="registerButton"
+              type="inner"
+              onClick={handleClick}
+              value="vertical"
+            >
+              Register
+            </Button>
+          </Row>
+        </Form.Item>
+        {show ? <Login /> : null}
+        {click ? <Register /> : null}
```

Comment:
why not just 1 state that renders either login or register ?

Reply:
can you show me an example with just one state? I can not do it just one.

Reply:
``` show ? <Login /> : <Register /> ```